### PR TITLE
Update electron to 11.1.1

### PR DESCRIPTION
**Pull request by Hinton**
Depends on: https://github.com/bitwarden/jslib/pull/247

### CODE REVIEW

**Review by cscharf — COMMENTED** [review]
Looks good, waiting on `jslib` update

File: package.json [hunk 1]
```diff
@@ -207,9 +207,9 @@
     "commander": "2.18.0",
     "core-js": "2.6.2",
     "duo_web_sdk": "git+https://github.com/duosecurity/duo_web_sdk.git",
-    "electron-log": "2.2.17",
-    "electron-store": "1.3.0",
-    "electron-updater": "4.2.0",
+    "electron-log": "4.3.0",
+    "electron-store": "6.0.1",
+    "electron-updater": "4.3.5",
```
Comment (kspearrin) [inline package.json:212]: I think some of these need to be updated in src/package.json

**Review by kspearrin — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Hinton) [reply package.json:212]: Correct, one of these days I'll write a test that verifies the versions are in sync ...

**Review by kspearrin — APPROVED** [review]

**Comment by kspearrin** [comment]
we might want to wait to merge this after this coming release, too.

**Review by cscharf — COMMENTED** [review]
Hi @Hinton , the following build fails with this update, `npm run dist:cli`

```bash
> bitwarden-directory-connector@0.0.0 pack:cli:lin /Users//source/bitwarden/directory-connector
> pkg . --targets linux-x64 --output ./dist-cli/linux/bwdc

> pkg@4.3.4
> Warning Cannot include directory %1 into executable.
  The directory must be distributed with executable as %2.
  node_modules/electron/dist
  path-to-executable/electron/dist
> Warning Cannot include file %1 into executable.
  The file must be distributed with executable as %2.
  node_modules/sliced/index.js
  path-to-executable/node_modules/sliced/index.js
> Warning Cannot include file %1 into executable.
  The file must be distributed with executable as %2.
  node_modules/deep-defaults/lib/index.js
  path-to-executable/node_modules/deep-defaults/index.js
> Warning Cannot include addon %1 into executable.
  The addon must be distributed with executable as %2.
  /Users//source/bitwarden/directory-connector/node_modules/keytar/build/Release/keytar.node
  path-to-executable/keytar.node
npmnpm ERR! code ELIFECYCLE
 ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! errno 2
npmnpm ERR! ERR! bitwarden-directory-connector@0.0.0 pack:cli:win: `pkg . --targets win-x64 --output ./dist-cli/windows/bwdc.exe`
npm bitwarden-directory-connector@0.0.0 pack:cli:mac: `pkg . --targets macos-x64 --output ./dist-cli/macos/bwdc`
 npmERR! ERR! Exit status 2
 Exit status 2
npmnpm  ERR!ERR! 
npm  
ERR!npm  Failed at the bitwarden-directory-connector@0.0.0 pack:cli:mac script.
ERR!npm ERR! Failed at the bitwarden-directory-connector@0.0.0 pack:cli:win script.
 This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users//.npm/_logs/2021-01-27T14_27_48_160Z-debug.log

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users//.npm/_logs/2021-01-27T14_27_48_160Z-debug.log
> Error! This experimental syntax requires enabling the parser plugin: 'optionalCatchBinding' (33:10)
  /Users//source/bitwarden/directory-connector/node_modules/electron-updater/node_modules/fs-extra/lib/empty/index.js
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! bitwarden-directory-connector@0.0.0 pack:cli:lin: `pkg . --targets linux-x64 --output ./dist-cli/linux/bwdc`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the bitwarden-directory-connector@0.0.0 pack:cli:lin script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users//.npm/_logs/2021-01-27T14_27_48_367Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! bitwarden-directory-connector@0.0.0 pack:cli: `npm run pack:cli:win | npm run pack:cli:mac | npm run pack:cli:lin`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the bitwarden-directory-connector@0.0.0 pack:cli script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users//.npm/_logs/2021-01-27T14_27_48_408Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! bitwarden-directory-connector@0.0.0 dist:cli: `npm run build:cli:prod && npm run clean:dist:cli && npm run pack:cli`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the bitwarden-directory-connector@0.0.0 dist:cli script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users//.npm/_logs/2021-01-27T14_27_48_446Z-debug.log
```

**Comment by MGibson1** [comment]
There are definitely commander update issues here. I'll push up a fix for that if no you don't mind, @Hinton

**Comment by Hinton** [comment]
@cscharf Builds should be working again. MacOS and Windows_gui fails due to missing code signing certs/keys.

**Review by cscharf — APPROVED** [review]

**Pull request merged**